src/screens/Embed/Embed.js

```diff
@@ -98,3 +158,20 @@ export default function Embed() {
 }
 // TODO
 function LoadingState() {}
+
+function findCountyByFips(fips) {
+  // NYC HACK.
+  if (['36047', '36061', '36005', '36081', '36085'].includes(fips)) {
+    fips = '36061';
+  }
+
+  const statesData = US_STATE_DATASET.state_county_map_dataset;
+  for (const state in statesData) {
+    const countiesData = statesData[state].county_dataset;
+    for (const county of countiesData) {
+      if (county.full_fips_code === fips) {
```

Comment:
consdier updating to String(county.full_fips_code) === String(fips)

Reply:
Sounds good. Done!